articles/deploying-fleet-on-aws-with-terraform.md

```diff
@@ -171,17 +171,17 @@ redis_cluster_members = toset([
 target_group_arn_suffix = "targetgroup/fleetdm/0f3bec83c8b02f58"
 ```
 
-We'll need some of these values in the next step.
+We'll need some of thes values in the next step.
```

Comment:
Should this say "these" or "the" in this sentence?

Reply:
This should say "these"

Reply:
@creedjax @chris-mcgillicuddy, I went ahead and made that change!